tracing/pom.xml

```diff
@@ -214,11 +214,6 @@
             <artifactId>mule-tracer-exporter-configuration-api</artifactId>
             <version>${project.version}</version>
         </dependency>
-        <dependency>
-            <groupId>org.mule.runtime</groupId>
-            <artifactId>mule-tracer-configuration-api</artifactId>
-            <version>${project.version}</version>
-        </dependency>
```

Comment:
What does this have to do with all the rest of the changes?

Reply:
This was wrongly placed there and was working by chance. Fixing odd things in this tests that cause them to fail in a more restrictive modularized environment.

Reply:
Sorry I missed the PR description